.github/CONTRIBUTING.md

```diff
@@ -1,7 +1,15 @@
 # Contributing
 
-When contributing to this repository, please first discuss the change you wish to make via issue,
-email, or any other method with the owners of this repository before making a change.
```

Comment:
We should keep this.

Reply:
I have actually split this line into two steps for better understanding and easy to follow contribution guideline.

Reply:
adding a sample contributing guideline for references -> [https://github.com/github/docs/blob/main/CONTRIBUTING.md](https://github.com/github/docs/blob/main/CONTRIBUTING.md)

Reply:
I have gone through this sample guideline document, and it seems that the points mentioned there are already covered in the three points I proposed below. These points will be easy to follow for any new contributors. 

What do you recommend? Should I elaborate the contribution guideline or keep it concise & easy to follow steps?

Reply:
Is this one need to resolve?